MC-29345: ENGCOM: Update documentation (devdocs) according to 2.4 (#6135)
magento/devdocs: src/_includes/contributor/2-4-submission.md

{:.bs-callout .bs-callout-info}
**v2.4 Contribution Notice:**

> We no longer accept pull requests to the v 2.3, 2.2, v2.1 or v2.0 release lines to focus all development efforts on v2.4.

Reviewer: ```suggestion
Magento is focusing development efforts on v2.4, so we are no longer accepting pull requests to the v2.3, v2.2, v2.1, and v2.0 release lines.
```
